samples/Ev.ServiceBus.Samples.Sender/Ev.ServiceBus.Samples.Sender.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
executables cannot have multiple target frameworks. it is OK to just put them in net8